openlibrary/tests/catalog/test_get_ia.py

```diff
@@ -6,6 +6,15 @@
 from openlibrary.catalog.marc.marc_xml import MarcXml
 from openlibrary.catalog.marc.marc_binary import MarcBinary, BadLength, BadMARC
 
+
+class MockResponse:
+    """MockResponse is used to pass the file back as a string instead of a file object.  This is because
+    this code was moved from urllib to requests."""
+    def __init__(self, data):
+        self.text = data
+        self.content = data
```

Comment:
Doesn't this set `.text` to bytes based on the expectations so the byte expectations will pass in test regardless of whether `content` or `text` is used? But then will fail in production when a real response is provided, where `text` will be a string and `content` will be bytes?

This looks like it breaks the test by making everything pass regardless.

Reply:
With the changes you are making you should not need to modify this test file, the request is requesting files, so it still makes sense to treat them as file objects.